camera_system/camera_system.py

```diff
@@ -16,6 +16,23 @@ def __init__(self, is_left_course: bool = True) -> None:
         """
         self.__set_is_left_course(is_left_course)
 
+    def start(self):
+        """ゲーム攻略を計画する."""
+        # 通信を確立する
+
+        # キャリブレーション
+
+        # 開始合図を受け取るまで待機
+
+        # ゲームエリア情報を生成する
+
+        # 計画する
+
+        # コマンドファイルを生成
+
+        # コマンドファイルを送信(システム外)
```

Comment:
コメントの先頭に`ToDo`を付けたら検索機能で簡単に探すことができるので、実装忘れが無いか探しやすくなると思います。
```suggestion
        # ToDo: 通信を確立する

        # ToDo: キャリブレーション

        # ToDo: 開始合図を受け取るまで待機

        # ToDo: ゲームエリア情報を生成する

        # ToDo: 計画する

        # ToDo: コマンドファイルを生成

        # ToDo: コマンドファイルを送信(システム外)
```